### ISSUE TITLE
listGroups doesn't mentions numbers

### ISSUE THREAD

**Issue by pzhuk** [AsamK/signal-cli#1158]
$ signal-cli --version
signal-cli 0.11.6

$ java -version
openjdk version "19.0.1" 2022-10-18
OpenJDK Runtime Environment (build 19.0.1+10-Ubuntu-1ubuntu122.04)

I am trying to observe a list of Signal group members. So using

signal-cli -o json -a <number> listGroups -d -g <group_id>

As result  I am getting at least 30% of members listed without number, but with only uuid mentioned.
  {
    "number": null,
    "uuid": "UUID_HAS_BEED_REDACTED_HERE"
  },

I was observing this group for 48 hours, tried to send messages, and of course using   signal-cli receive periodically - none of that helps, members stay with null numbers.
When using Signal Desktop - issue is not observed, all numbers are available.

Problem:  signal-cli provides uuid, but doesn't give any interface to use this uuid. Not possible to send message, or remove member of the group, or anything else.
Basically I am stuck with null phone and profile uuid which can't be used in any way.

Suggestion - provide some uuid based interface?

**Comment by AsamK** [comment]
It's not documented yet, but you can use the uuid in all the places where signal-cli expects a phone number. So you can send messages to it and remove group members.
The phone numbers of group members can only be seen when these members send a message. The group metadata itself just contains the uuid.
My guess would be that the Signal-Desktop instance has been active longer and observed more phone numbers. Or is signal-cli the primary device, then it might be something else.

There are upcoming changes in Signal for completely hiding the phone number and usernames, I wanted to wait for those before really committing to a way to handle only uuid contacts.

**Comment by nojonqs** [comment]
> It's not documented yet, but you can use the uuid in all the places where signal-cli expects a phone number. So you can send messages to it and remove group members. The phone numbers of group members can only be seen when these members send a message. The group metadata itself just contains the uuid. My guess would be that the Signal-Desktop instance has been active longer and observed more phone numbers. Or is signal-cli the primary device, then it might be something else.

It seems to not be possible to get the contact information with `listContacts` just with the uuid. Is there any other way to manually get that and therefore "restore" the broken contact you get from `listGroups`?
